code/src/components/TodoSummary.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { todos } from "reducers/todos";
+import styled from "styled-components";
+import swal from "sweetalert";
+import { Button } from "lib/Button";
+
+export const TodoSummary = () => {
+  const list = useSelector((store) => store.todos.list);
+  const doneList = list.items.filter((item) => item.done);
+  const dispatch = useDispatch();
+  const [trueOrFalse, setTrueOrFalse] = useState(false);
+
+  const onRemoveClicked = (event) => {
+    swal({
+      title: "Are you sure?",
+      text: "Once deleted, you will not be able to recover your todo's",
+      icon: "warning",
+      buttons: true,
+      dangerMode: true,
+    }).then((willDelete) => {
+      if (willDelete) {
+        dispatch(todos.actions.removeAllTodos());
+        swal("1, 2, 3! Your todo's are now todont's", {
+          icon: "success",
+        });
+      } else {
+        swal("Phew!", "Your todo's are safe!");
+      }
+    });
+  };
```

Comment:
Nice add-on! I really like that you're exploring extra things. So valuable. You are a true creative developer 👩‍🎨